usrsctplib/netinet/sctp_indata.c

```diff
@@ -1745,7 +1745,7 @@ sctp_process_a_data_chunk(struct sctp_tcb *stcb, struct sctp_association *asoc,
 		 * empty data chunk.
 		 */
 		op_err = sctp_generate_no_user_data_cause(tsn);
-		stcb->sctp_ep->last_abort_code = SCTP_FROM_SCTP_INDATA + SCTP_LOC_14;
+		stcb->sctp_ep->last_abort_code = SCTP_FROM_SCTP_INDATA + SCTP_LOC_15;
```

Comment:
@taylor-b is this change done to resolve uniqueness violation, because `SCTP_LOC_14` is used here:
https://github.com/sctplab/usrsctp/blob/32020d6f44d405ef4ea3fecb52bc8a392aa8a6c4/usrsctplib/netinet/sctp_indata.c#L1592
?

Reply:
Yes, as long as I was adding more SCTP_LOCs I went ahead and made the rest ordered and unique.